crates/sui-indexer-alt/src/lib.rs

```diff
@@ -318,3 +339,83 @@ impl Indexer {
         Ok(Some(watermark))
     }
 }
+
+pub async fn start_indexer(
+    indexer_config: IndexerConfig,
+    db_config: DbConfig,
+    sequential_pipeline_config: SequentialPipelineConfig,
+    // If true, the indexer will bootstrap from genesis.
+    // Otherwise it will skip the pipelines that rely on genesis data.
+    with_genesis: bool,
```

Comment:
Is it worth doing this the other way? I.e. infer whether we need to bootstrap genesis based on whether the pipelines we're about to run need it or not? Seems like that would be less error-prone.

Reply:
Is there a better way to infer that other than hardcode the list of pipelines that need genesis somewhere?